package.json

```diff
@@ -7,7 +7,7 @@
   "license": "MIT",
   "repository": "github:facebook/react-native",
   "engines": {
-    "node": ">=12"
+    "node": ">=14"
```

Comment:
This will create a warning or error for users of RN with Node 12. Is it okay to make the breaking change yet?

Reply:
Also, why 14 and not 16?

Reply:
@NickGerleman I think it's ok to make it a breaking change, folks should be on current LTS (so 16).

@Saadnajmi it's 14 because on CircleCI we also have a job that tests against the previous LTS (14) hence why the requirement. You can see me bumping that prevlts in this PR

Reply:
> @NickGerleman I think it's ok to make it a breaking change, folks should be on current LTS (so 16).

Agree here. Plus the current Node version (non lts) is 17.
I think it's totally ok to drop Node 12 (maybe we should add a note in the Changelog for this).

Reply:
yeah let me change the changelog entry

Reply:
Seems like the changelog entry didn't sync properly, we might need to update this separately when building the changelog

Reply:
@ShikaSD interesting - is this because I changed the text of the PR body after the import process was started?

Reply:
I think Nico landed the previous version internally, so we didn't re-import the description

Reply:
gotcha - makes sense.